lib/spack/spack/cmd/module.py

```diff
@@ -27,24 +27,23 @@
 import collections
 import os
 import shutil
-import sys
+import spack.modules
 
 import spack.cmd
-
 from llnl.util import filesystem, tty
 from spack.cmd.common import arguments
-from spack.modules import module_types
 
 description = "manipulate module files"
 section = "environment"
 level = "short"
 
 
-# Dictionary that will be populated with the list of sub-commands
-# Each sub-command must be callable and accept 3 arguments :
-# - mtype : the type of the module file
-# - specs : the list of specs to be processed
-# - args : namespace containing the parsed command line arguments
+#: Dictionary that will be populated with the list of sub-commands
+#: Each sub-command must be callable and accept 3 arguments:
+#:
+#:   - mtype : the type of the module file
+#:   - specs : the list of specs to be processed
+#:   - args : namespace containing the parsed command line arguments
```

Comment:
OOoooooh!!! I was wondering how to do this! Can you point me to the Sphinx docs on this feature?